### PR TITLE
Documentation/dev: Update the debugging.md file.

### PR DESCRIPTION
**Pull request by timflannagan1**
This would attempt to add more clarity to the documentation, and highlight any important information as needed.

### CODE REVIEW

File: Documentation/dev/debugging.md [hunk 1]
```diff
@@ -4,35 +4,35 @@ Debugging metering can be fairly difficult if you do not know how to directly in
 Below, we detail how you can connect and query Presto and Hive, as well as view the dashboards of the Presto and HDFS components.
 For debugging issues surrounding the metering-ansible-operator, see the [ansible-operator](#metering-ansible-operator) section.
 
-All of the follow commands assume you've set the `METERING_NAMESPACE` environment variable to the namespace your Metering installation is located in:
+**Note**: All of the follow commands assume you've set the `METERING_NAMESPACE` environment variable to the namespace your Metering installation is located in:
```
Comment (bentito) [inline Documentation/dev/debugging.md]: typo: s/follow/following/

**Comment by bentito** [comment]
One typo

File: Documentation/dev/debugging.md (hunk 1 above)
Comment (timflannagan1) [reply Documentation/dev/debugging.md]: @bentito done.

**Pull request merged**